runtime/src/main/java/org/corfudb/runtime/view/LayoutManagementView.java

```diff
@@ -97,18 +99,16 @@ public void handleFailure(IReconfigurationHandlerPolicy failureHandlerPolicy,
      * Bootstraps the new node with the current layout.
      * This action is invoked as a part of the add node workflow which requires the new node
      * to be added to the cluster to be bootstrapped with the existing layout of this cluster.
-     * This bootstraps the Layout and the Management server with the existing layout which
-     * initiates failure handling capabilities on the management server.
+     * This bootstraps the Layout server with the existing layout.
      *
      * @param endpoint New node endpoint.
-     * @return Completable Future which completes when the node's layout and management servers are bootstrapped.
+     * @return Completable Future which completes when the node's layout is bootstrapped.
      */
     public CompletableFuture<Boolean> bootstrapNewNode(String endpoint) {
 
         // Bootstrap the to-be added node with the old layout.
         Layout layout = new Layout(runtime.getLayoutView().getLayout());
         return runtime.getLayoutView().bootstrapLayoutServer(endpoint, layout)
-                .thenCompose(result -> runtime.getManagementView().bootstrapManagementServer(endpoint, layout))
```

Comment:
Why have we removed this?

Reply:
Because the AddNode workflow performs the RestoreRedundancy action. Since this action is performed by the node itself, the orchestrator should be selected on the current node. For orchestrator, the management server should have an existing management layout already present.